feature-group-definitions/read-write-pseudos.yml

```
@@ -1,4 +1,5 @@
-name: ":read and :write"
+name: ":read-only and :read-write"
+description: The `:read-only` and `:read-write` CSS pseudo-classes match elements that are read-only or read-write, respectively, such as when `<input>` or `<textarea>` elements have the `readonly` attribute set or not set, respectively.
```

The sentence is a bit long. Trying this:

```suggestion
description: The `:read-only` and `:read-write` CSS pseudo-classes match elements that are read-only or read-write, respectively. For example, `:read-only` matches `<input>` and `<textarea>` elements with the `readonly` attribute.
```